### ISSUE TITLE
download stopped due to file identified as malware or spam

### ISSUE THREAD

**Issue by dd-han** [prasmussen/gdrive#234]
console show:

```
Failed to download file: googleapi: got HTTP response code 403 with body: {
 "error": {
  "errors": [
   {
    "domain": "global",
    "reason": "cannotDownloadAbusiveFile",
    "message": "This file has been identified as malware or spam and cannot be downloaded."
   }
  ],
  "code": 403,
  "message": "This file has been identified as malware or spam and cannot be downloaded."
 }
}
```

anyway to bypass this file or force download file ?

**Comment by szunyi** [comment]
you can download this file from browser?

**Comment by dd-han** [comment]
@szunyi Yes, I can force the download after confirming the warning on the browser.

But here I can't.

**Comment by Bomel** [comment]
Is there a solution for this? Because I have the same problem.

**Comment by dd-han** [comment]
@Bomel no, I manually download file to my computer by web browser

then upload to headless machine by ssh

**Comment by s4y** [comment]
FYI: There's a parameter, `acknowledgeAbuse=true`, that can be passed to bypass this warning. The API docs ask that software explicitly get permission from the user before passing it, and passing it for a non-blocked file is an error.

**Comment by mbenlioglu** [comment]
User can be prompted to acknowledge abusive file to forcefully download

**Comment by ijat9917** [comment]
> User can be prompted to acknowledge abusive file to forcefully download

How? in terminal?

**Comment by dkb007** [comment]
How to fix it in code? can anyone tell me

**Comment by dd-han** [comment]
@dkb007 I switch to Rclone for my purpose

**Comment by Pakos091** [comment]
> > User can be prompted to acknowledge abusive file to forcefully download
> 
> 
> 
> How? in terminal?

Did you find out?

**Comment by Proff-Roman** [comment]
Все решается просто.
В файле /gdrive/drive/download.go
находим:
`res, err := self.service.Files.Get(f.Id).SupportsTeamDrives(true).Context(ctx).Download()`
заменяем на:
`res, err := self.service.Files.Get(f.Id).AcknowledgeAbuse(true).SupportsTeamDrives(true).Context(ctx).Download()`

Как компилировать - смотрим тут - [https://github.com/prasmussen/gdrive/issues/392#issuecomment-472186479](url)
